Tasks/ServiceFabricDeploy/task.loc.json

```diff
@@ -83,6 +83,15 @@
       "groupname": "advanced",
       "helpMarkDown": "ms-resource:loc.input.help.compressPackage"
     },
+    {
+      "name": "useDiffPackage",
+      "type": "boolean",
+      "label": "ms-resource:loc.input.label.useDiffPackage",
+      "defaultValue": "true",
```

Comment:
You are making this default as true. Wont this affect the existing users of the task? Should the diff be the default? Adding @RoopeshNair  for his opinion here. Is it safe to change the deployment method for existing users?

Reply:
Deploying with delta changes makes sense to be defaulted. I am wondering if users have worked around this and now if the behavior changes, will it impact them?

Reply:
@sachinma @RoopeshNair For a new deployment, diffpackage equals to fullpackge so it is the same with current behavior. For a upgrade, the only difference is that diffpackage won't include non-upgraded service packages, which is tested by us and works fine.